src/provider/provider_level_zero.c

```diff
@@ -57,23 +57,29 @@ static UTIL_ONCE_FLAG ze_is_initialized = UTIL_ONCE_FLAG_INIT;
 static bool Init_ze_global_state_failed;
 
 static void init_ze_global_state(void) {
-    // check if Level Zero shared library is already loaded
-    // we pass 0 as a handle to search the global symbol table
+// check if Level Zero shared library is already loaded
```

Comment:
I guess you could move this comment below the `ifdef`

Reply:
Done, good point